src/routers/semester.ts

```diff
@@ -1,24 +1,51 @@
 import express from 'express';
+import { create, read, update, del } from '../operators/semesterOperations';
 
 const semesterRouter = express.Router();
 
-semesterRouter.get('/', (req, res) => {
-	console.log('Get Semester');
-	res.send('Get semesterRouter');
+const errorResponse = 'Semester not found.';
```

Comment:
this is like, so small, but do we have a naming convention for constants like these? (eg, `ERROR_RESPONSE`) :]

Reply:
good catch - pushed a change 😄